### DISCUSSION TITLE
Ho to setting max of row for RichEditor?

### DISCUSSION THREAD

**Discussion by altimusin** [filamentphp/filament#5265] [Help]
Ho to setting max default of row for RichEditor ?
I want to set 10 rows as default.

`RichEditor::make('otherinfo')->label('Other Information')->columnSpan('full')`

![image](https://user-images.githubusercontent.com/43429317/207915286-e46a36e6-1611-44d1-8e1e-04aed32f06f8.png)

**Comment by ArtMin96** [comment]
I think you have to do it with CSS because it is not a textarea.

**Reply by RX-77IT** [reply]
For future readers, this can be done with something like
```php
RichEditor::make('otherinfo')
->extraInputAttributes(['style' => 'min-height: 20rem; max-height: 50vh; overflow-y: auto;'])
```
